frontend/src/wizards/Placement/PlacementBinding.tsx

```diff
@@ -102,29 +107,30 @@ function PlacementBinding(props: { bindingSubjectKind: string; bindingSubjectApi
 
 function Subject() {
     const subject = useItem() as IPlacementSubject
+    const { t } = useTranslation()
     return (
         <Fragment>
             <Select
                 path="kind"
-                label="Subject kind"
+                label={t('Subject kind')}
                 required
-                options={['PolicySet', 'Policy']}
+                options={[t('PolicySet'), t('Policy')]}
```

Comment:
Another case where we need to distinguish value/label. But in this case, we are referring to a "kind" which should probably not be translated anyway. Translation team should know not to translate something like "PolicySet"... hard to say how they would know not to translate "Policy". In any case, never rely on a translation for proper functioning of the code though.

Reply:
updated to [`{label:'t(...)', value: '...'}`]
Translation team said to wrap anything questionable wrt kube resources, and they will use their discretion.